index.html

```diff
@@ -0,0 +1,48 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title> CLARENCE PORTFOLIO</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;700;900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/073688fc60.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <!-- Navigation -->
+
+  <header>
+    <a clsss="my-logo" href="#">Clarence</a>
+    <nav>
+      <i class="fa-solid fa-bars"></i>
+    </nav>
+  </header>
+
+  <!-- Hero Section -->
+
+  <section class="hero">
+    <div class="home-header">
+      <h1>Hey There.<br>I'm Clarence</h1>
+      <h3>I'm a software developer</h3>
```

Comment:
Please I'd suggest you use a `p` tag for this paragraph as it is semantically correct to use the `p` tag for all paragraphs and to enable screen readers and SEO technologies identify that it is not an actual header 😄